### PR TITLE
feat: AccountPickers component

### PR DESCRIPTION
**Pull request by gmolki**
## Summary

This PR creates a new AccountPickers component with all needed behaviours for connecting to an account and switching between networks. 
Adding this component deprecates WalletPicker component, so it has been removed.

## Demo

Videos with all variants of the component:

### Twentysix

https://github.com/user-attachments/assets/50901b31-77ab-4b9f-995d-66c580b3a100

### Aleph

https://github.com/user-attachments/assets/7f0c2775-86e1-4fe1-b2d4-307c8e994dc3

### CODE REVIEW

File: src/components/modules/AccountPickers/cmp.tsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+            <NetworkSelector
+              networks={networks}
+              selectedNetwork={selectedNetwork}
+              onSelectNetwork={handleSwitchNetwork}
+            />
+          </StyledPicker>,
+          document.body,
+        )}
+      {displayCondensedPicker &&
+        createPortal(
```
Comment (amalcaraz) [inline src/components/modules/AccountPickers/cmp.tsx:160]: Use our `<Portal>` component from `modules/Portal/` instead to make sure this component is 100% compatible with nextjs SSR

**Review by amalcaraz — CHANGES_REQUESTED** [review]
Just spotted a few details.
Really great work overall!

File: src/components/modules/AccountPickers/cmp.tsx (hunk 1 above)
Comment (gmolki) [reply src/components/modules/AccountPickers/cmp.tsx]: Sure! Didn't know about that!
Thanks for spotting 🙌🏻

**Review by amalcaraz — APPROVED** [review]
LGTM

**Pull request merged**